cherrypy/lib/static.py

```diff
@@ -314,6 +314,13 @@ def staticdir(section, dir, root='', match='', content_types=None, index='',
     branch = request.path_info[len(section) + 1:]
     branch = urllib.parse.unquote(branch.lstrip(r'\/'))
 
+    # On Windows requesting a file in sub-dir of the staticdir results
+    # in mixing of delimiter styles, eg: C:\static\js/script.js
+    # Python normally converts this but not when the staticdir is
+    # supplied in long-path notation, eg: \\?\C:\static\js/script.js
+    if os.name == 'nt':
+        branch = branch.replace('/', '\\')
```

Comment:
Please use `r'\'`

Reply:
That's not possible as a function parameter:
```
E       branch = branch.replace('/', r'\')
E                                        ^
E   SyntaxError: EOL while scanning string literal
```

Reply:
Oh.. I see